search-app/src/cmr/search/services/query_execution/has_granules_or_cwic_results_feature.clj

```diff
@@ -10,7 +10,8 @@
    [cmr.elastic-utils.search.es-index :as common-esi]
    [cmr.redis-utils.config :as redis-config]
    [cmr.redis-utils.redis-cache :as redis-cache]
-   [cmr.search.data.elastic-search-index :as idx]))
+   [cmr.search.data.elastic-search-index :as idx]
```

Comment:
Remove [cmr.search.data.elastic-search-index :as idx] as it isn't needed anymore.  

The linter should run and this line should show up as yellow in the VSCode telling you that it isn't used.